src/aocharmovie.cpp

```diff
@@ -26,32 +26,23 @@ AOCharMovie::AOCharMovie(QWidget *p_parent, AOApplication *p_ao_app)
 void AOCharMovie::play(QString p_char, QString p_emote, QString p_emote_prefix,
                        bool p_visible)
 {
-  QString target_path;
-  QStringList f_paths{
-      ao_app->get_character_path(p_char, p_emote_prefix + p_emote), // .gif
-      ao_app->get_character_path(p_char, p_emote),                  // .png
-      ao_app->get_theme_variant_path("placeholder"),                // .gif
-      ao_app->get_theme_path("placeholder"),                        // .gif
-      ao_app->get_default_theme_path("placeholder")                 // .gif
-  };
-
-  for (auto &f_file : f_paths)
-  {
-    bool found = false;
-    for (auto &ext : QStringList{".webp", ".apng", ".gif", ".png"})
-    {
-      QString fullPath = ao_app->get_case_sensitive_path(f_file + ext);
-      found = file_exists(fullPath);
-      if (found)
+  // Asset lookup order
```

Comment:
Aren't these comments just a duplication of `find_asset_path`s documentation?

Reply:
Not exactly, they are plain English descriptions of the 3 types of places where AOCharMovie assets are looked for.

Reply:
Not exactly, they are plain English descriptions of the three types of places where AOCharMovie assets are looked for.